dashboard/app/models/lesson.rb

```diff
@@ -344,7 +357,8 @@ def summarize(include_bonus_levels = false, for_edit: false)
         description_student: description_student,
         description_teacher: description_teacher,
         unplugged: unplugged,
-        lessonEditPath: get_uncached_edit_path
+        lessonEditPath: get_uncached_edit_path,
+        lessonStartPath: start_url
```

Comment:
(Nit) Since this is a url it might be better named `lessonStartUrl`

Reply:
+1

Reply:
excellent point- updated!